src/libs/connectorLibs/saveFiles.js

```diff
@@ -305,14 +307,23 @@ async function createFile(client, entry, options, method, fileId) {
 
   let fileDocument
   if (method === 'create') {
-    fileDocument = await cozy.files.create(toCreate, createFileOptions)
+    const clientResponse = await client.save({
+      _type: 'io.cozy.files',
+      type: 'file',
+      data: toCreate,
+      ...createFileOptions
+    })
+    fileDocument = clientResponse.data
   } else if (method === 'updateById') {
     log.debug(`replacing file for ${entry.filename}`)
-    fileDocument = await cozy.files.updateById(
-      fileId,
-      toCreate,
-      createFileOptions
-    )
+    const clientResponse = client.save({
+      _id: file._id,
+      _rev: file._rev,
+      _type: 'io.cozy.files',
+      data: toCreate,
+      ...createFileOptions
+    })
+    fileDocument = clientResponse.data
```

Comment:
This might be out of scope of this PR, but the `method` param to determine if it's a create or update feels a bit weird and unnecessary. We could simply save the file, and let cozy-client deal with the create or update, which is based on the existence or not of the  `_rev` attribute : https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L792-L797

Reply:
Yeah we already speak about that with @doubleface in the case of save. But we want to erase cozy-client-js first, propagate into the future ccc_lib before beginning to optimized the code

Reply:
> But we want to erase cozy-client-js first, propagate into the future ccc_lib before beginning to optimized the code


Ok, but I feel like it would have been more efficient to do all the API improvements at the same time :)

Anyways, maybe we should list all the possible improvements in a trello card? Especially since there are some crossed remarks with @Crash-- in the other PR: https://github.com/cozy/cozy-flagship-app/pull/555